docs/features/docker_compose.md

```diff
@@ -7,28 +7,165 @@ This is intended to be useful on projects where Docker Compose is already used
 in dev or other environments to define services that an application may be
 dependent upon.
 
+<<<<<<< HEAD
 You can override Testcontainers-go's default behaviour and make it use a
+=======
+## Using `docker-compose` directly
+
+Because `docker-compose` v2 is implemented in Go it's possible for _testcontainers-go_ to
+use [`github.com/docker/compose`](https://github.com/docker/compose) directly and skip any process execution/_
+docker-compose-in-a-container_ scenario.
+The `ComposeStack` API exposes this variant of using `docker-compose` in an easy way.
+
+### Basic examples
+
+Use the convenience `NewDockerComposeAPI(...)` constructor which creates a random identifier and takes a variable number
```

Comment:
Double checking the docs I found that we did not update them when we decided to rename the function. Could you verify the example code is executable? 🙏 
```suggestion
Use the convenience `NewDockerCompose(...)` constructor which creates a random identifier and takes a variable number
```